x-pack/plugins/timelines/public/components/t_grid/integrated/index.tsx

```diff
@@ -232,7 +227,7 @@ const TGridIntegratedComponent: React.FC<TGridIntegratedProps> = ({
     loading,
     { events, loadPage, pageInfo, refetch, totalCount = 0, inspect },
   ] = useTimelineEvents({
-    alertConsumers: SECURITY_ALERTS_CONSUMERS,
```

Comment:
We use this table for the hosts page as well, so just passing this value in on the alerts table and leaving it out on the host page